dropshot/Cargo.toml

```diff
@@ -9,14 +9,14 @@ repository = "https://github.com/oxidecomputer/dropshot/"
 
 [dependencies]
```

Comment:
bump to version 0.3.0? (I bumped to 0.2.0 when I added support for `tags` in the `endpoint` metadata

Reply:
Done.  (Looks like I'll need to bump it again when I merge with #16.)

Reply:
Meh... I now think we should just bump it when we push to crates.io, but up to you